modules/index-service/src/main/java/org/opencastproject/index/service/resources/list/api/ResourceListProvider.java

```diff
@@ -56,5 +56,14 @@
    * @param listName the name of the list
    * @return if the results should be translated
    */
-  boolean isTranslatable(String listName);
+  public abstract boolean isTranslatable(String listName);
+
+  /**
+   * Returns the key of the default value in the list if customized, else null.
+   *
+   * @return key of default value or null //TODO Or is blank better?
```

Comment:
Making this method concrete hides the decision from the Impl programmer. IMHO, it would be better to leave this method as an API which requires the Implementation programmer to consider what makes the most sense for the type of Implementation that implements this API. 

The program that calls this method cannot make assumptions if this will be blank or null because the Impl programer can always override the method to return a blank or null.

Reply:
I agree with the second part. Regarding the first part of your comment: I decided to provide a default (hehe) implementation for getDefault() since a lot of the other list providers currently have no need for a default and I didn't want to bloat the code with a lot of "...return null". This is debatable of course.